sigma9.css

```
@@ -1,3 +1,10 @@
+@charset "utf-8";
+ 
+@import url('http://www.scp-wiki.net/local--files/component:theme/font-bauhaus.css');
```

Should use the new domain:
```suggestion
@import url('http://www.scpwiki.com/local--files/component:theme/font-bauhaus.css');
```

---

We decided on using the direct link instead:
```suggestion
@import url('http://scp-wiki.wdfiles.com/local--files/component%3Atheme/font-bauhaus.css');
```